File: php/3rd_week/latihanpraktikum3_php/Processor.php
<?php

class Processor{
    // implementasi class processor

    // atribut
    private $name = "";
    private $procPrice = 0;

    //constructor
    function __construct()
    {
    }

    //set and get atribut
    function setName($name){
        $this->name = $name;
    }

    function setProcPrice($price){
        $this->procPrice = $price;
    }

    function getName(){
        return $this->name;
    }

    function getProcPrice(){
        return $this->procPrice;
    }

    // method print
    function printProc(){
        echo "-Data Processor". "<br/>";
        echo " Nama Processor : ". $this->name. "<br/>";
        echo " Harga Processor : ". $this->procPrice. "<br/>";
        echo "--------------------------------------------". "<br/>";
    }

    //destructor
    function __destruct()
    {
    }
}
?>

File: python/3rd_week/latihanpraktikum3_py/Processor.py
class Processor:
    # implementasi class processor

    # atribut
    name = ""
    procPrice = 0

    def __init__(self):
        pass

    def setName(self, name):
        self.name = name

    def getName(self):
        return self.name    
    
    def setProcPrice(self, procPrice):
        self.procPrice = procPrice
    
    def getProcPrice(self):
        return self.procPrice

    def printProc(self):
        print("-Data Processor")
        print(f" Nama Processor  : {self.name}")
        print(f" Harga Processor : {self.procPrice}")
        print("--------------------------------------")
    
